bundles/paikkatietoikkuna/coordinateconversion/resources/css/coordinateconversion.css

```diff
@@ -0,0 +1,322 @@
+    display:inline-block;
+    width:12px;
+    height:12px;
+    margin:-1px 4px 0 0;
+    vertical-align:middle;
+    cursor:pointer;
+    -moz-border-radius:  50%;
+    border-radius:  50%;
+    border: 0.5px solid black;
+}
+input[type="radio"] + label span {
+     background-color:whitesmoke;
+}
+input[type="radio"]:checked + label span{
+     background-color:black;
+}
+
+input[type="radio"] + label span,
+input[type="radio"]:checked + label span {
+  -webkit-transition:background-color 0.4s linear;
+  -o-transition:background-color 0.4s linear;
+  -moz-transition:background-color 0.4s linear;
+  transition:background-color 0.4s linear;
+}
+
+/*
+*TABLE
+*/
+#oskari-coordinate-table {
+    height: 80%;
+    overflow-y: scroll;
+}
+.oskari-table-content {
+    float:left;
+    width: 20em !important;
+    height: 20em;
+    overflow: auto;
+    margin-top: 0px;
+    border: 1px solid rgba(255,255,255,0.3);
+}
+.oskari-table-content-target {
+    float:right;
+    width: 20em !important;
+    height: 20em;
+    overflow: hidden;
+    margin-top: 0px;
+    border: 1px solid rgba(255,255,255,0.3);
+}
+.oskari-table-header {
+    background-color:rgba(37, 184, 193, 0.8);
+    width: 20em !important;
+}
+#oskari-tbl-header {
+    margin-bottom: 0;
+    table-layout: fixed;
+    word-wrap: break-word;
+}
+.target {
+    padding-left:2.5em;
+}
+th {
+  padding: 20px 8px;
+  text-align: left;
+  font-weight: 500;
+  font-size: calc(0.55rem + 0.28vh);
+  color: #fff;
+  text-transform: uppercase;
+  width: 33% !important;
+}
+td {
+    background: #FAFAFA;
+    text-align: center;
+}
+.cell {
+    height: 1em;
+}
+.cell.control {
+    border: inherit; 
+    border-bottom: 1px solid black;
+    display: none; 
+    position: relative;
+}
+
+/*
+* TABLE HOVER
+*/
+.hoverable {
+  background: white;
+  border-radius:3px;
+  border-collapse: collapse;
+  height: 15em;
+  margin: auto;
+  max-width: 600px;
+  padding:5px;
+  width: 100%;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.1);
+  animation: float 5s infinite;
+}
+.hoverable td { 
+    padding:7px; border:#4e95f4 1px solid;
+    width:33% !important;
+}
+/* Define the hover highlight color for the table row */
+.hoverable tr:hover {
+  background:#4E5066;
+  color:#FFFFFF;
+  border-top: 1px solid #22262e;
+  border-bottom: 1px solid #22262e;
+}
+tr:hover td {
+  background:#4E5066;
+  color:#FFFFFF;
+  border-top: 1px solid #22262e;
+  border-bottom: 1px solid #22262e;
+}
+.hoverable tr:hover .cell.control { 
+    /* float: right; */
+    /* display: inline-block; */
+}
+
+.removerow {
+    border-radius: 50%;
+    height:1.3em;
+    width:1.4em;
+    background-image: url('../images/icon-close.png');
+    right:-5em;
+    position: absolute;
+}
+.select {
+    float:right;
+}
+.dropdown_title {
+    float: left;
+    font-weight: bold;
+    word-break: break-word;
+    max-width:12em !important;
+}
+.infolink {
+    float:right;
+    background-image: url('../images/icon-info.png');
```

Comment:
You can use icon-info class for the component so you don't need to add this image to the bundle resources. The icon is part of the base set of icons.